server/src-lib/Hasura/Tracing.hs

```diff
@@ -197,42 +202,68 @@ instance MonadTrace m => MonadTrace (ExceptT e m) where
   currentReporter = lift currentReporter
   attachMetadata = lift . attachMetadata
 
+-- | encode Byte String to 16 character hex string
+encodeBstoHex :: BS.ByteString -> BS.ByteString
```

Comment:
Should `encodeBstoHex` return back `Text` instead of ByteString?

Because I think Text type makes more sense, since having ByteString might suggest that we might have characters outside the UTF-8 range.

Reply:
Having it return `Text` is the right thing.